.rubocop.yml

```diff
@@ -1,6 +1,11 @@
 require: rubocop-rspec
 inherit_from: .rubocop_todo.yml
 
+AllCops:
+  Exclude:
+    - 'spec/pb-ruby/goodbye_pb.rb'
+    - vendor/bundle/**/*
```

Comment:
Why is Rubocop looking in `vendor/bundle`?

Reply:
I dunno, but it is without this excludes statement. I think it's just looking in all the subfolders.
It actually manifests in rubocop complaining about the ruby version of some dependency oddly enough
https://app.circleci.com/pipelines/github/Gusto/grpc-web-ruby/174/workflows/e6045431-6d32-47a7-84ca-68a6318aa268/jobs/330/steps

 A similar excludes statement exists in ZP.